packages/core/package.json

```diff
@@ -14,9 +14,7 @@
   "devDependencies": {
     "@awaitjs/express": "^0.3.0",
     "@eth-optimism/contracts": "^0.4.5",
-    "@eth-optimism/message-relayer": "^0.1.8",
```

Comment:
the optimism bot (still wip) that transfers messages cross-chain will still need this package.

Reply:
Re-added!